Pilot.xcodeproj/project.pbxproj

```diff
@@ -1241,13 +1245,17 @@
 			buildSettings = {
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```

Comment:
we should probably pull in the latest version of https://github.com/jspahrsummers/xcconfigs/commits/master as it probably contains a bunch of these.